tests/e2e/specs/pacbio/create_pacbio_library_from_samples_page.cy.js

```diff
@@ -19,6 +19,7 @@ describe('Pacbio library creation from sample', () => {
     cy.get('#samples-table').contains('td', '5')
     cy.get('#samples-table').first().click()
     cy.get('#pacbioLibraryCreate').click()
+    cy.get('#tag-set-input').select('Sequel_16_barcodes_v3')
```

Comment:
a check to see if tag-input is disabled before selecting tag set may be a good thing to test here.